### ISSUE TITLE
Tests using a mobile or tablet device

### ISSUE THREAD

**Issue by Angelsaur** [alan-turing-institute/AutSPACEs#59]
### Summary

You’ll need to start by doing some checks using a mobile or tablet device to check that your website is usable on these types of device.

### Deadline 

20/02/2021

### What needs to be done?

 - Check the page orientation

You’ll need to take a few of your sample pages and check they respond properly when you try to change the page orientation.

You can do this by loading up a page on a mobile or tablet device and turning the screen from portrait to landscape a few times. The page should respond to fit that view, rather than staying locked in place.

 - Check you can navigate using just one finger

Some users have limited dexterity or range of movement, which means it’s important that they do not have to use complex gestures that involve more than one finger to navigate your website on their mobile or tablet device. There needs to be a simple alternative that they can use that only requires one finger.

Go through your sample content pages and check whether it’s possible to navigate using just one finger. Try things like double-tapping to zoom in and out of the page and holding or hovering your finger over an element to highlight it.

It’s worth trying to complete a couple of forms using these techniques, too.

 - Check you’re not relying on complex motions
Some more interactive websites or apps require users to do things like shake or tilt their device to complete an action.

This will not apply to most websites, but if it applies to yours then you need to make sure there’s an alternative for users who cannot move their device in these ways.

Check that none of your sample pages require any of these types of interactions - and if they do, that there’s another way of completing that task.
